### PR TITLE
Update quickstartquide

### PR DESCRIPTION
**Pull request by ranjithwingrider**
Signed-off-by: ranjithwingrider ranjith.raveendran@openebe.io

### CODE REVIEW

File: docs/quickstartguide.md [hunk 1]
```diff
@@ -14,31 +14,22 @@ Following are the steps to get started with OpenEBS on a Kubernetes cluster.
 
 2. Kubernetes 1.9.7+ is installed
```
Comment (mahebbar) [inline docs/quickstartguide.md]: installed.

File: docs/quickstartguide.md [hunk 2]
```diff
@@ -14,31 +14,22 @@ Following are the steps to get started with OpenEBS on a Kubernetes cluster.
...
 
-3. You can install OpenEBS either through stable helm chart or OpenEBS helm chart or by using OpenEBS operator/kubectl.  
-
-   **Note:** Currently OpenEBS version 0.7 is supported only via the [OpenEBS operator/kubectl](/docs/next/installation.html#install-openebs-using-kubectl) and [OpenEBS helm Charts](/docs/next/installation.html#install-openebs-using-openebs-helm-charts) .
+3. You can install OpenEBS either through stable helm chart or by using OpenEBS operator/kubectl.  
 
+  
       ### Using a Stable Helm Chart:
```
Comment (mahebbar) [inline docs/quickstartguide.md]: remove :

File: docs/quickstartguide.md [hunk 3]
```diff
@@ -14,31 +14,22 @@ Following are the steps to get started with OpenEBS on a Kubernetes cluster.
...
 
    [Setup RBAC for helm tiller](/docs/next/installation.html#helm) and install the chart from Kubernetes stable and deploy the storage class templates.
```
Comment (mahebbar) [inline docs/quickstartguide.md]: Kubernetes stable what?

File: docs/quickstartguide.md [hunk 4]
```diff
@@ -14,31 +14,22 @@ Following are the steps to get started with OpenEBS on a Kubernetes cluster.
...
-   helm repo add openebs-charts https://openebs.github.io/charts/
-   helm repo update
-   helm install openebs-charts/openebs
-   ```
-
-      ### Using OpenEBS Operator (kubectl):
+    
+    ### Using OpenEBS Operator (kubectl):
 
    Ensure that you run the `kubectl` commands with cluster admin context. The installation will involve creating a new Service Account and assigning to OpenEBS components.
```
Comment (mahebbar) [inline docs/quickstartguide.md]: and assigning it to OpenEBS components.

**Comment by ranjithwingrider** [comment]
Already changes are reflected

**Pull request closed without merging**